src/main/java/com/airlines/dao/CrewMemberDao.java

```diff
@@ -0,0 +1,12 @@
+package com.airlines.dao;
+
+import com.airlines.entity.CrewMember;
+
+public interface CrewMemberDao {
+
+    void save(CrewMember account);
```

Comment:
why object type is CrewMember and parameter type is "Account"?

Reply:
fixed in 45b9bb5fd29cdf8936e2dab8daa2a25ad85a7b59